src/command_modules/azure-cli-datalake/azure/cli/command_modules/datalake/analytics/_params.py

```diff
@@ -0,0 +1,38 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+from azure.cli.core.commands import \
+    (register_cli_argument, CliArgumentType)
+from azure.cli.core.commands.parameters import \
+    (tags_type, location_type, resource_group_name_type,
+     get_resource_name_completion_list,
+     enum_choice_list)
+
+from azure.mgmt.datalake.analytics.account.models.data_lake_analytics_account_management_client_enums \
+    import(FirewallState,
+           TierType,
+           FirewallAllowAzureIpsState)
+
+from azure.mgmt.datalake.analytics.job.models.data_lake_analytics_job_management_client_enums \
+    import(CompileMode)
+
+# pylint: disable=line-too-long
+# ARGUMENT DEFINITIONS
+
+datalake_analytics_name_type = CliArgumentType(help='Name of the Data Lake Analytics account.', options_list=('--account_name',), completer=get_resource_name_completion_list('Microsoft.DataLakeAnalytics/accounts'), id_part=None)
+
+
+# PARAMETER REGISTRATIONS
+register_cli_argument('datalake analytics', 'account_name', datalake_analytics_name_type, options_list=('--account', '-n'))
+register_cli_argument('datalake analytics account', 'resource_group_name', resource_group_name_type, completer=None, validator=None)
+register_cli_argument('datalake analytics account create', 'location', location_type)
+register_cli_argument('datalake analytics account create', 'tags', tags_type)
+register_cli_argument('datalake analytics account create', 'tier', help='The desired commitment tier for this account to use.', **enum_choice_list(TierType))
+register_cli_argument('datalake analytics account update', 'tags', tags_type)
+register_cli_argument('datalake analytics account update', 'tier', help='The desired commitment tier for this account to use.', **enum_choice_list(TierType))
```

Comment:
And that won't try to bind this argument to any other commands (or nested commands) that don't have tags/tier?